tool.html

```diff
@@ -3,19 +3,7 @@
   <head>
 		<script src='angular.js'></script>
 		<script src="canvas.js"></script>
-		<script>
-				// Angular.js code.
-				function MyController($document, $http, $interval) {
-					this.$onInit = (function() {
-						this.intervalService_ = $interval;
-						this.httpService_ = $http;
-						this.document_ = $document;
-						this.fastaFileName;		// Setting on controller to share the file name with QUAL file uploader.
-						this.qualFileName;		// Setting on controller to share the file name with FASTA file uploader.
-						this.reset_();
-					}).bind(this);
-				}		
-		</script>
+		<script src='client.js'></script>

```

Comment:
I have moved the client-side logic to client.js (https://github.com/gbelwariar/Microbiome-Diversity-Inspector/blob/master/client.js)

Reply:
That's great.